contentcuration/contentcuration/frontend/channelEdit/components/edit/InheritAncestorMetadataModal.vue

```diff
@@ -192,6 +192,19 @@
     },
     methods: {
       ...mapActions('contentNode', ['updateContentNode']),
+      /**
+       * @public
+       */
+      checkInheritance() {
+        if (this.allFieldsDesignatedByParent || !this.parentHasInheritableMetadata) {
+          // If all fields have been designated by the parent, or there is nothing to inherit,
+          // automatically continue
+          this.handleContinue();
+        } else {
+          // Wait for the data to be updated before showing the dialog
+          this.closed = false;
```

Comment:
Just noticed that the only way `closed = false` is if we execute this line of code, that means that `closed = false` just happens if `this.allFieldsDesignatedByParent || !this.parentHasInheritableMetadata` is false. So I think we can remove these conditions from the `active` computed prop [here](https://github.com/rtibbles/studio/blob/5827048670fab9b24663e1642c606d16223c9e0e/contentcuration/contentcuration/frontend/channelEdit/components/edit/InheritAncestorMetadataModal.vue#L116-L118) as they are redundant, and just left active true if `parent !== null && !this.closed`. 

This makes the logic much more easier to understand

Reply:
Actually I just noticed that if `!this.parent` => `inheritableMetadataItems = {}` => `parentHasInheritableMetadata = false` => `this.allFieldsDesignatedByParent || !this.parentHasInheritableMetadata` will always be true => we will never execute the `this.closed = false`. So this is also a redundant condition that can be removed from the `active` prop :eyes:.

Reply:
Yes, I think this would work if we start `closed` as `true` in the initialized data, that way it doesn't show up until it's ready?

Reply:
Oh I just saw that we already initialize this value to true https://github.com/rtibbles/studio/blob/5827048670fab9b24663e1642c606d16223c9e0e/contentcuration/contentcuration/frontend/channelEdit/components/edit/InheritAncestorMetadataModal.vue#L98